chain-signatures/node/src/mesh/connection.rs

```diff
@@ -143,6 +154,10 @@ impl Pool {
 
     async fn set_potential_participants(&self, participants: &Participants) {
         *self.potential_connections.write().await = participants.clone();
+        tracing::debug!(
+            "Pool set potential participants to {:?}",
+            self.potential_connections.read().await.keys_vec()
+        );
     }
```

Comment:
I'd vote for getting rid of these `setting the participants` logs as the contract state already logs the participants